app/models/repo.js

```diff
@@ -130,18 +130,47 @@ Repo.reopen({
     return array;
   }.property(),
 
+  crons: function() {
+    var array, builds, id;
+    id = this.get('id');
+    builds = this.store.filter('build', {
+      event_type: 'cron',
+      repository_id: id
+    }, function(b) {
+      return b.get('repo.id') + '' === id + '' && b.get('eventType') === 'cron';
+    });
+    array = ExpandableRecordArray.create({
+      type: 'build',
+      content: Ember.A([])
+    });
+    array.load(builds);
+    id = this.get('id');
+    array.observe(builds);
+    return array;
+  }.property(),
```

Comment:
I see now that this almost totally mirrors the implementation of `pullRequests`. I understand more now why you chose to implement it this way, since it’s a pattern that already exists. If it works, you can just keep it the way it is and I’ll refactor this and the `pullRequests` code to be more intelligible and concise. I’m just getting to know the codebase so sorry for complaining about code you’re just inheriting!